src/applications/static-pages/simple-forms/40-0247/App.js

```diff
@@ -0,0 +1,176 @@
+          can’t return them to you.
+        </p>
+        <p>
+          <va-link
+            href="https://www.cem.va.gov/CEM/hmm/discharge_documents.asp"
+            text="Review our list of military discharge documents"
+          />
+        </p>
+        <h3>Option 3: Upload your application</h3>
+        <p>
+          First, complete the Presidential Memorial Certificate Request Form (VA
+          Form 40-0247).
+        </p>
+        <p>
+          <va-link
+            href="/find-forms/about-form-40-0247"
+            text="Get VA Form 40-0247 to download"
+          />
+        </p>
+        <p>
+          Then, upload your completed application and supporting documents on
+          the AccessVA website using the QuickSubmit tool.
+        </p>
+        <p>
+          If it’s your first time signing into this tool, you’ll need to
+          register first. After you’ve registered, you can upload your
+          application and documents online.
+        </p>
+        <p>
+          <va-link
+            href="https://eauth.va.gov/accessva/?cspSelectFor=quicksubmit"
+            text="Upload your application on AccessVA"
+          />
+        </p>
+        <h3>Option 4: Submit your application in person</h3>
+        <p>You can apply at any VA regional office.</p>
+        <p>
+          <va-link
+            href="/find-locations/?facilityType=benefits"
+            text="Find your nearest VA regional office"
+          />
+        </p>
+        <h3>Option 5: Fax your application</h3>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <h3>Option 1: Upload your application</h3>
+      <p>
+        First, complete the Presidential Memorial Certificate Request Form (VA
+        Form 40-0247).
+      </p>
+      <p>
+        <va-link
+          href="/find-forms/about-form-40-0247"
+          text="Get VA Form 40-0247 to download"
+        />
+      </p>
+      <p>
+        Then, upload your completed application and supporting documents on the
+        AccessVA website using the QuickSubmit tool.
+      </p>
+      <p>
+        If it’s your first time signing into this tool, you’ll need to register
+        first. After you’ve registered, you can upload your application and
+        documents online.
+      </p>
+      <p>
+        <va-link
+          href="https://eauth.va.gov/accessva/?cspSelectFor=quicksubmit"
+          text="Upload your application on AccessVA"
+        />
+      </p>
+      <h3>Option 2: Mail your application</h3>
+      <p>
+        First, complete the Presidential Memorial Certificate Request Form (VA
+        Form 40-0247).
+      </p>
+      <p>
+        <va-link
+          href="/find-forms/about-form-40-0247"
+          text="Get VA Form 40-0247 to download"
+        />
+      </p>
+      <p>
+        Mail your application, along with copies of the Veteran’s death
+        certificate and DD214 or other discharge documents, to this address:
+      </p>
+      <p>
+        NCA FP Evidence Intake Center
```

Comment:
Same question as above. Is it possible to use the address component here for these 3 lines? @tlei123 @tbaker1026

https://design.va.gov/components/address-block

Reply:
@megzehn , yes; className="va-address-block" added to the <p> tag.  It'll looks like an [address block](https://design.va.gov/storybook/?path=/docs/components-address-block--default-story) on your page.